RecoJets/JetProducers/python/ak4PFJets_cfi.py

```diff
@@ -17,7 +17,8 @@
     )
 
 ak4PFJetsPuppi = ak4PFJets.clone(
-    src = cms.InputTag("puppi")
+    src = cms.InputTag("particleFlow"),
+    applyPuppiWeight=cms.bool(True)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.